src/main/java/org/jfrog/hudson/pipeline/common/types/deployers/GradleDeployer.java

```diff
@@ -18,16 +20,42 @@
     private Boolean deployIvyDescriptors;
     private String ivyPattern = "[organisation]/[module]/ivy-[revision].xml";
     private String artifactPattern = "[organisation]/[module]/[revision]/[artifact]-[revision](-[classifier]).[ext]";
+    private String repoValidationMessage = "Deployer configuration must contain either 'repo' field, or 'releaseRepo' and 'snapshotRepo' fieds.";
```

Comment:
I suggest rephrasing the message to:
The Deployer should be set with either 'repo' or both 'releaseRepo' and 'snapshotRepo'.

Also - 
private String repoValidationMessage --> private **static final** String repoValidationMessage